cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,12 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.get(':nth-child(1) > .form-control').type('tester111@test.net');
```

Comment:
It is a bad practice to hardcode credentials in code. Consider moving these to environment variables or a secure storage and accessing them from there.